scala test build error

[scala test build error](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_sanity.external_x86-64_linux/45/) on Feb 11, 2022. It looks like `SBT_VERSION ` is not set properly.


```
00:12:56.289       [exec] Step 9/25 : ARG SBT_VERSION=1.2.8
00:12:56.289       [exec]  ---> Running in b42c10d1a3ce
00:12:56.289       [exec] Removing intermediate container b42c10d1a3ce
00:12:56.289       [exec]  ---> cb882d3f38a0
00:12:56.289       [exec] Step 10/25 : ENV SBT_VERSION=$SBT_VERSION
00:12:56.289       [exec]  ---> Running in 1209c73aab54
00:12:56.289       [exec] Removing intermediate container 1209c73aab54
00:12:56.289       [exec]  ---> e88e5a6d070c
00:12:56.289       [exec] Step 11/25 : ENV SBT_HOME=/opt/sbt
00:12:56.289       [exec]  ---> Running in dc92fa9caad3
00:12:56.705       [exec] Removing intermediate container dc92fa9caad3
00:12:56.705       [exec]  ---> e42c4b61b719
00:12:56.705       [exec] Step 12/25 : RUN wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz) 	&& wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz.asc](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz.asc) 	&& curl "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | gpg --import 	&& gpg --verify sbt-${SBT_VERSION}.tgz.asc sbt-${SBT_VERSION}.tgz 	&& tar -zvxf sbt-${SBT_VERSION}.tgz -C /opt/ 	&& rm -f sbt-${SBT_VERSION}.tgz 	&& rm -f sbt-${SBT_VERSION}.tgz.asc
00:12:56.705       [exec]  ---> Running in 91975379f98e
00:12:57.143       [exec] [91m--2022-02-14 07:49:30--  https://piccolo.link/sbt-1.2.8.tgz
00:12:57.143       [exec] [0m[91mResolving piccolo.link (piccolo.link)... [0m[91m150.239.84.182, 52.116.121.9
00:12:57.556       [exec] Connecting to piccolo.link (piccolo.link)|150.239.84.182|:443... [0m[91mconnected.
00:12:57.556       [exec] [0m[91mWARNING: cannot verify piccolo.link's certificate, issued by ‘CN=R3,O=Let's Encrypt,C=US’:
00:12:57.556       [exec]   Issued certificate has expired.
00:13:01.078       [exec] HTTP request sent, awaiting response... [0m[91m502 Bad Gateway
00:13:01.078       [exec] 2022-02-14 07:49:34 ERROR 502: Bad Gateway.
00:13:01.078       [exec] 
00:13:01.078       [exec] The command '/bin/sh -c wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz) 	&& wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz.asc](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz.asc) 	&& curl "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | gpg --import 	&& gpg --verify sbt-${SBT_VERSION}.tgz.asc sbt-${SBT_VERSION}.tgz 	&& tar -zvxf sbt-${SBT_VERSION}.tgz -C /opt/ 	&& rm -f sbt-${SBT_VERSION}.tgz 	&& rm -f sbt-${SBT_VERSION}.tgz.asc' returned a non-zero code: 8
00:13:01.078       [exec] [0m
00:13:01.078  
00:13:01.078  BUILD FAILED
00:13:01.078  /home/jenkins/workspace/Test_openjdk11_j9_sanity.external_x86-64_linux/aqa-tests/TKG/scripts/build_test.xml:95: The following error occurred while executing this line:
00:13:01.078  /home/jenkins/workspace/Test_openjdk11_j9_sanity.external_x86-64_linux/aqa-tests/external/build.xml:42: The following error occurred while executing this line:
00:13:01.078  /home/jenkins/workspace/Test_openjdk11_j9_sanity.external_x86-64_linux/aqa-tests/external/scala/build.xml:25: The following error occurred while executing this line:
00:13:01.078  /home/jenkins/workspace/Test_openjdk11_j9_sanity.external_x86-64_linux/aqa-tests/external/build.xml:87: exec returned: 8
00:13:01.078  
00:13:01.078  Total time: 11 minutes 32 seconds
00:13:01.078  1
00:13:01.078  make[1]: *** [compile.mk:39: compile] Error 1
00:13:01.078  make[1]: Leaving directory '/home/jenkins/workspace/Test_openjdk11_j9_sanity.external_x86-64_linux/aqa-tests/TKG'
00:13:01.078  make: *** [makefile:67: compile] Error 2
```

[The last successful build](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_sanity.external_x86-64_linux/44/) was Feb 4, 2022

Same error in Adopt Grinder: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3622/console
```
00:29:29.429       [exec] Step 9/25 : ARG SBT_VERSION=1.2.8
00:29:29.429       [exec]  ---> Running in 7a974e0228cf
00:29:29.429       [exec] Removing intermediate container 7a974e0228cf
00:29:29.429       [exec]  ---> 412d5ecf89dd
00:29:29.429       [exec] Step 10/25 : ENV SBT_VERSION=$SBT_VERSION
00:29:29.429       [exec]  ---> Running in 25637c80adee
00:29:29.429       [exec] Removing intermediate container 25637c80adee
00:29:29.429       [exec]  ---> d0747da7e7e7
00:29:29.429       [exec] Step 11/25 : ENV SBT_HOME=/opt/sbt
00:29:29.429       [exec]  ---> Running in b24fc19b0bef
00:29:29.429       [exec] Removing intermediate container b24fc19b0bef
00:29:29.429       [exec]  ---> eacf8b7dc1b7
00:29:29.429       [exec] Step 12/25 : RUN wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz) 	&& wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz.asc](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz.asc) 	&& curl "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | gpg --import 	&& gpg --verify sbt-${SBT_VERSION}.tgz.asc sbt-${SBT_VERSION}.tgz 	&& tar -zvxf sbt-${SBT_VERSION}.tgz -C /opt/ 	&& rm -f sbt-${SBT_VERSION}.tgz 	&& rm -f sbt-${SBT_VERSION}.tgz.asc
00:29:29.429       [exec]  ---> Running in 30ea041b4ac1
00:29:30.077       [exec] [91m--2022-02-14 23:11:27--  https://piccolo.link/sbt-1.2.8.tgz
00:29:30.077       [exec] [0m[91mResolving piccolo.link (piccolo.link)... [0m[91m52.116.121.9, 150.239.84.182
00:29:30.077       [exec] Connecting to piccolo.link (piccolo.link)|52.116.121.9|:443... [0m[91mconnected.
00:29:30.077       [exec] [0m[91mWARNING: cannot verify piccolo.link's certificate, issued by ‘CN=R3,O=Let's Encrypt,C=US’:
00:29:30.077       [exec]   Issued certificate has expired.
00:29:33.063       [exec] HTTP request sent, awaiting response... [0m[91m502 Bad Gateway
00:29:33.063       [exec] 2022-02-14 23:11:31 ERROR 502: Bad Gateway.
00:29:33.063       [exec] 
00:29:33.373       [exec] The command '/bin/sh -c wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz) 	&& wget --no-check-certificate --no-cookies [https://piccolo.link/sbt-${SBT_VERSION}.tgz.asc](https://piccolo.link/sbt-$%7BSBT_VERSION%7D.tgz.asc) 	&& curl "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | gpg --import 	&& gpg --verify sbt-${SBT_VERSION}.tgz.asc sbt-${SBT_VERSION}.tgz 	&& tar -zvxf sbt-${SBT_VERSION}.tgz -C /opt/ 	&& rm -f sbt-${SBT_VERSION}.tgz 	&& rm -f sbt-${SBT_VERSION}.tgz.asc' returned a non-zero code: 8
00:29:33.373       [exec] [0m
00:29:33.373  
00:29:33.373  BUILD FAILED
00:29:33.373  /home/jenkins/workspace/Grinder/aqa-tests/TKG/scripts/build_test.xml:95: The following error occurred while executing this line:
00:29:33.373  /home/jenkins/workspace/Grinder/aqa-tests/external/scala/build.xml:25: The following error occurred while executing this line:
00:29:33.373  /home/jenkins/workspace/Grinder/aqa-tests/external/build.xml:87: exec returned: 8
```

--- a/external/dockerfile_functions.sh
+++ b/external/dockerfile_functions.sh
@@ -258,8 +258,8 @@ print_sbt_install() {
           "\nENV SBT_VERSION=\$SBT_VERSION" \
           "\nENV SBT_HOME=/opt/sbt" \
           "\n\n# Install SBT" \
-          "\nRUN wget --no-check-certificate --no-cookies https://piccolo.link/sbt-\${SBT_VERSION}.tgz \\" \
-          "\n\t&& wget --no-check-certificate --no-cookies https://piccolo.link/sbt-\${SBT_VERSION}.tgz.asc \\" \
+          "\nRUN wget --no-check-certificate --no-cookies https://github.com/sbt/sbt/releases/download/v\${SBT_VERSION}/\${SBT_VERSION}.tgz \\" \
+          "\n\t&& wget --no-check-certificate --no-cookies https://github.com/sbt/sbt/releases/download/v\${SBT_VERSION}/sbt-\${SBT_VERSION}.tgz.asc \\" \
           "\n\t&& curl \"https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823\" | gpg --import \\" \
           "\n\t&& gpg --verify sbt-\${SBT_VERSION}.tgz.asc sbt-\${SBT_VERSION}.tgz \\" \
           "\n\t&& tar -zvxf sbt-\${SBT_VERSION}.tgz -C /opt/ \\" \

--- a/external/dockerfile_functions.sh
+++ b/external/dockerfile_functions.sh
@@ -258,7 +258,7 @@ print_sbt_install() {
           "\nENV SBT_VERSION=\$SBT_VERSION" \
           "\nENV SBT_HOME=/opt/sbt" \
           "\n\n# Install SBT" \
-          "\nRUN wget --no-check-certificate --no-cookies https://github.com/sbt/sbt/releases/download/v\${SBT_VERSION}/\${SBT_VERSION}.tgz \\" \
+          "\nRUN wget --no-check-certificate --no-cookies https://github.com/sbt/sbt/releases/download/v\${SBT_VERSION}/sbt-\${SBT_VERSION}.tgz \\" \
           "\n\t&& wget --no-check-certificate --no-cookies https://github.com/sbt/sbt/releases/download/v\${SBT_VERSION}/sbt-\${SBT_VERSION}.tgz.asc \\" \
           "\n\t&& curl \"https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823\" | gpg --import \\" \
           "\n\t&& gpg --verify sbt-\${SBT_VERSION}.tgz.asc sbt-\${SBT_VERSION}.tgz \\" \

--- a/external/scala/test.properties
+++ b/external/scala/test.properties
@@ -1,6 +1,6 @@
 github_url="https://github.com/scala/scala.git"
 script="scala-test.sh"
 home_path=""
-tag_version="v2.13.2"
-sbt_version="1.2.8"
+tag_version="v2.13.8"
+sbt_version="1.6.2"
 ubuntu_packages="git wget tar curl gpg gpg-agent"

